Resources/doc/mapping.rst

```diff
@@ -0,0 +1,207 @@
+Mapping
+=======
+
+Elasticsearch bundle requires mapping definition in order for it to work with index.
+
+Mapping configuration
+---------------------
+
+Here’s an example of configuration with tokenizer’s, filters and analyzer’s definitions:
+
+.. code:: yaml
+
+    elasticsearch:
+        connections:
+            default:
+                hosts:
+                    - { host: 127.0.0.1:9200 }
+                index_name: acme_index
+                settings:
+                    number_of_shards: 2
+                    number_of_replicas: 0
+                    index:
+                        refresh_interval: -1
+                    analysis:
+                        tokenizer:
+                            pathTokenizer:
+                                type : path_hierarchy
+                                buffer_size: 2024
+                                skip: 0
+                                delimiter: /
+                        filter:
+                            incremental_filter:
+                                type: edge_ngram
+                                min_gram: 1
+                                max_gram: 20
+                        analyzer:
+                            pathAnalyzer:
+                                type: custom
+                                tokenizer: pathTokenizer
+                            urlAnalyzer:
+                                type: custom
+                                tokenizer: keyword
+                                filter: [lowercase]
+                            incrementalAnalyzer:
+                                type: custom
+                                tokenizer: standard
+                                filter:
+                                    - lowercase
+                                    - incremental_filter
+        managers:
+            default:
+                connection: default
+                debug: true
+
+In the settings node user may choose a specific configuration for that connection with index settings.
+
+In managers configuration ``mappings`` is optional. If there are no mappings defined it will look up through all bundles ``Document`` folders. If ``debug`` is set to true, profiler and logging will be enabled (by default it is set to false).
```

Comment:
through all bundles'; although rephrasing would probably be better: it will look up through all ``Document`` folders cointained in a bundle.